projects/ad9656/zcu102/Makefile

```diff
@@ -0,0 +1,29 @@
+####################################################################################
+## Copyright 2018(c) Analog Devices, Inc.
+## Auto-generated, do not modify!
+####################################################################################
+
+PROJECT_NAME := ad9656_zcu102
+
+M_DEPS += ../common/ad9656_bd.tcl
+M_DEPS += ../../common/zcu102/zcu102_system_constr.xdc
+M_DEPS += ../../common/zcu102/zcu102_system_bd.tcl
+M_DEPS += ../../../library/xilinx/common/ad_iobuf.v
+M_DEPS += ../../../library/util_cdc/sync_bits.v
+M_DEPS += ../../../library/jesd204/scripts/jesd204.tcl
+M_DEPS += ../../../library/common/util_pulse_gen.v
+M_DEPS += ../../../library/common/ad_bus_mux.v
+
+LIB_DEPS += axi_clkgen
+LIB_DEPS += axi_dmac
+LIB_DEPS += axi_sysid
+LIB_DEPS += jesd204/ad_ip_jesd204_tpl_adc
+LIB_DEPS += jesd204/axi_jesd204_rx
+LIB_DEPS += jesd204/jesd204_rx
+LIB_DEPS += sysid_rom
+LIB_DEPS += util_pack/util_cpack2
+LIB_DEPS += util_pack/util_upack2
```

Comment:
Upack is probably not needed, as there is no DAC in the design

Reply:
Upack library insertion deleted from the Makefile